### PR TITLE
Cosmos fix readme

### PR DESCRIPTION
**Pull request by kushagraThapar**
* Cosmos fix readme: https://github.com/Azure/azure-sdk-for-java/issues/11643
* Follows general guidelines : https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/template/azure-sdk-template/README.md

To see how the final readme will look like -  https://github.com/kushagraThapar/azure-sdk-for-java/blob/cosmos_fix_readme/sdk/cosmos/azure-cosmos/README.md

### CODE REVIEW

File: sdk/cosmos/azure-cosmos/README.md [hunk 1]
```diff
@@ -1,133 +1,149 @@
 ﻿# Azure CosmosDB Client Library for Java
 
-[![Maven Central](https://img.shields.io/maven-central/v/com.microsoft.azure/azure-cosmos.svg)](https://search.maven.org/artifact/com.microsoft.azure/azure-cosmos)
```
Comment (moderakh) [inline sdk/cosmos/azure-cosmos/README.md:3]: link to the sdk on maven is good to have. any reason it is moved? can we add it back?

File: sdk/cosmos/azure-cosmos/README.md [hunk 2]
```diff
@@ -269,28 +258,44 @@ We have added the ability to modify the default Direct TCP options utilized by t
...
+the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to 
+do this once across all repos using our CLA.
+
+This project has adopted the [Microsoft Open Source Code of Conduct][coc]. For more information see the [Code of Conduct FAQ][coc_faq] 
+or contact [opencode@microsoft.com][coc_contact] with any additional questions or comments.
+
+<!-- LINKS -->
+[source_code]: src
+[cosmos_introduction]: https://docs.microsoft.com/en-us/azure/cosmos-db/
+[api_documentation]: https://azuresdkdocs.blob.core.windows.net/$web/java/azure-cosmos/4.0.1-beta.3/index.html
```
Comment (moderakh) [inline sdk/cosmos/azure-cosmos/README.md:283]: end users may search for the word "javadoc" can we add that as a keyword as well?

File: sdk/cosmos/azure-cosmos/README.md (hunk 1 above)
Comment (kushagraThapar) [reply sdk/cosmos/azure-cosmos/README.md:3]: I will add it back, no specific reason to remove it.

File: sdk/cosmos/azure-cosmos/README.md (hunk 2 above)
Comment (kushagraThapar) [reply sdk/cosmos/azure-cosmos/README.md:283]: Yes, will do.

File: sdk/cosmos/azure-cosmos/README.md (hunk 1 above)
Comment (kushagraThapar) [reply sdk/cosmos/azure-cosmos/README.md:3]: Done.

File: sdk/cosmos/azure-cosmos/README.md (hunk 2 above)
Comment (kushagraThapar) [reply sdk/cosmos/azure-cosmos/README.md:289]: Done.

**Pull request merged**